library/src/main/java/com/google/android/exoplayer/hls/HlsChunkSource.java

```diff
@@ -541,6 +553,17 @@ private boolean allPlaylistsBlacklisted() {
     return true;
   }
 
+  private void evaluatePlaylistBlacklistedTimestamps()
+  {
+    long currentTime = SystemClock.elapsedRealtime();
+    for (int i = 0; i < mediaPlaylistBlacklistFlags.length; i++) {
+      if (mediaPlaylistBlacklistFlags[i] && currentTime - mediaPlaylistBlacklistedTimeMs[i] > DEFAULT_MAX_TIME_MEDIA_PLAYLIST_BLACKLISTED_MS) {
```

Comment:
nit - Line too long. Please line break before the && to get it back to within 100 chars.